jota/src/test/java/org/iota/jota/IotaAPITest.java

```diff
@@ -78,384 +106,619 @@
+                .thenThrow(new ArgumentException(Constants.INVALID_ADDRESSES_INPUT_ERROR));
 
-    @Test
-    @Tag("IntegrationTest")
-    public void shouldGetInclusionStates() throws ArgumentException {
-        GetInclusionStateResponse res = iotaAPI.getInclusionStates(new String[]{TEST_HASH});
-        assertThat("States should be an array of booleans", res.getStates(), IsNull.notNullValue());
-        assertTrue(res.getStates()[0], "Hash should have been seen as confirmed");
-    }
-    
-    @Test
-    @Tag("IntegrationTest")
-    public void shouldIsConfirmed() throws ArgumentException {
-        GetInclusionStateResponse res = iotaAPI.isConfirmed(new String[]{TEST_HASH});
-        assertThat("States should be an array of booleans", res.getStates(), IsNull.notNullValue());
-        assertTrue(res.getStates()[0], "Hash should have been seen as confirmed");
+        ArgumentException assertThrows = assertThrows(ArgumentException.class, () ->
+                iotaAPIMock.prepareTransfers(TEST_SEED2, 2, transfers, null, inputs, null, false));
+
+        assertEquals(Constants.INVALID_ADDRESSES_INPUT_ERROR, assertThrows.getMessage(), "Message should say that there is not enough balance");
     }
 
     @Test
-    @Tag("IntegrationTest")
     public void shouldFindTransactionObjects() throws ArgumentException {
-        List<Transaction> ftr = iotaAPI.findTransactionObjectsByAddresses(TEST_ADDRESSES);
-        assertThat("findTransactionObjectsByAddresses should not return null on failure", ftr, IsNull.notNullValue());
+        List<Transaction> transactions = Collections.singletonList(new Transaction.Builder()
+                .branchTransaction(BRANCH_TRANSACTION)
+                .trunkTransaction(TRUNK_TRANSACTION)
+                .address(TEST_ADDRESSES[0])
+                .signatureFragments(SIGNATURE_FRAGMENTS)
+                .tag(TEST_TAG)
+                .timestamp(Instant.now().getEpochSecond())
+                .nonce("WBTKE9RPRMHDJGITAENBMGDKWXQ")
+                .obsoleteTag(TEST_TAG)
+                .attachmentTimestamp(ATTACHMENT_TIMESTAMP)
+                .attachmentTimestampLowerBound(ATTACHMENT_TIMESTAMP_LOWER_BOUND)
+                .attachmentTimestampUpperBound(ATTACHMENT_TIMESTAMP_UPPER_BOUND)
+                .hash("AKNHNOQTIMRCETAWNHZXEHLUERYUHGCGEVFHO9JF9AL9J9CXDDUVAKBYPJMXWEJCPUJXRNXCQBRWBQ999")
+                .currentIndex(0)
+                .lastIndex(2)
+                .bundle(BUNDLE)
+                .value(1)
+                .build());
 
-        assertFalse(ftr.isEmpty(), "findTransactionObjectsByAddresses should find multiple transactions");
+        when(iotaAPIMock.findTransactionObjectsByAddresses(TEST_ADDRESSES)).thenReturn(transactions);
 
+        List<Transaction> transactionObjectsByAddresses = iotaAPIMock.findTransactionObjectsByAddresses(TEST_ADDRESSES);
+
+        assertNotNull(transactionObjectsByAddresses);
```

Comment:
Codacy found an issue: [JUnit assertions should include a message](https://app.codacy.com/manual/Iota-java/iota-java/pullRequest?prid=6319461)